spec/models/user_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] Great job so far 👏🏻 However let's make sure User Model specs have **Validation** tests to validate the presence of the user table columns (name, post_counter,..) Example:

  ```ruby
  describe 'validations' do
    it { should validate_presence_of(:name) }
  end
  ```